pkg/volume/local/local.go

```diff
@@ -207,6 +213,17 @@ func (m *localVolumeMounter) SetUpAt(dir string, fsGroup *int64) error {
 	if !notMnt {
 		return nil
 	}
+	if fsGroup != nil {
+		fsGroupSame, fsGroupOld, err := isSamefsGroup(m.globalPath, fsGroup)
```

Comment:
@msau42 I think it is fair, especially since they can still use default and not specify fsGroup for the POD. If they need specific Gid, then it would make sense to create it.